Framework/Algorithms/test/GeneratePythonFitScriptTest.h

```diff
@@ -115,7 +148,7 @@ class GeneratePythonFitScriptTest : public CxxTest::TestSuite {
   std::string createExpectedSequentialScriptText() const {
     std::string expectedScript;
 
-    expectedScript += "# A python script generated to perform a sequential fit\n";
+    expectedScript += "# A python script generated to perform a sequential or simultaneous fit\n";
```

Comment:
Ditto on this, could we just have a reference .py file, and load that instead
This makes it much easier to compare a reference to a generated file by saving both out and doing a diff, rather than trying to read `\n` all over the place